app/assets/stylesheets/welcome.scss

```diff
@@ -160,6 +160,29 @@ span.list-group-item.active {
   color: #e573fc;
   font-size: 10px;
 }
+
 .url-list {
   padding-bottom: 200px;
 }
+
+#flash_notice {
+  &.error {
+    border: solid 4px #900;
+          background-color: #ff6565;
+        }
+  &.notice {
+    border: solid 4px #26722D;
```

Comment:
Color literals like `#26722D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#26722D` should be written as `#26722d`<br>Properties should be ordered background-color, border